indico/modules/events/reminders/notifications_test.py

```diff
@@ -14,9 +14,6 @@
 from indico.web.flask.templating import get_template_module
 
 
-pytest_plugins = 'indico.modules.events.timetable.testing.fixtures'
```

Comment:
This breaks the citadel plugin which still uses `create_entry`:

https://github.com/indico/indico-plugins/blob/1794b8baeccb598772ffcd868d7d30fd2df1c85b/citadel/indico_citadel/schemas_test.py#L24

Reply:
if you're going to open a PR on that repo: make sure to reference this PR (by URL) in the description of the other PR; that way the CI will automatically use indico from your branch instead of master to run the tests